zntrack/core/data_classes.py

```diff
@@ -15,6 +15,10 @@
 from dataclasses import dataclass, field
```

Comment:
Again, up to you but some of the type hints are missing in the parameters of methods here.

Reply:
I think this should be resolved in f176e4c 
I appreciate any feedback on missing docstrings on the main code.